External/Plugins/ASCompletion/Context/IASContext.cs

```diff
@@ -207,7 +206,14 @@ public interface IASContext
         /// </summary>
         bool UnsetOutOfDate();
 
+        /// <summary>
+        /// Returns the default value for the given type
+        /// </summary>
+        /// <param name="type"></param>
+        /// <returns>Default value</returns>
         string GetDefaultValue(string type);
+
+        List<string> DecomposeTypes(List<string> types);
```

Comment:
I would say that using `IEnumerable<string>`, at least as the returning type (but probably the argument too) is preferrable. I don't see you have any specific need for `List`.

Reply:
Fixed